src/openssl/lib_ssl.cr

```diff
@@ -2,8 +2,8 @@ require "./lib_crypto"
 
 {% begin %}
   lib LibSSL
-    OPENSSL_110 = {{ `command -v pkg-config > /dev/null && pkg-config --atleast-version=1.1.0 libssl || printf %s false`.stringify != "false" }}
-    OPENSSL_102 = {{ `command -v pkg-config > /dev/null && pkg-config --atleast-version=1.0.2 libssl || printf %s false`.stringify != "false" }}
+    OPENSSL_110 = {{ LibCrypto::LIBRESSL_VERSION.nil? && (LibCrypto::OPENSSL_VERSION >= 0x1010100) }}
```

Comment:
Just `{{ LibCrypto::OPENSSL_110 }}` should work, surely?

Reply:
Yeah, my bad.